client/src/containers/Authenticate/index.js

```diff
@@ -64,15 +69,32 @@ export default function Authenticate({ children }) {
           const name = googleUserProfile.getName();
           const email = googleUserProfile.getEmail();
           const id_token = googleUser.getAuthResponse().id_token;
-          dispatch({
-            type: 'LOGIN_SUCCESS',
-            payload: {
-              user: {
-                name,
-                email,
-                id_token,
-              },
-            },
+
+          const newHeaders = new Headers();
+          newHeaders.append('Authorization', `Token ${id_token}`);
+          newHeaders.append('Content-Type', 'application/json');
+          newHeaders.append('Accept', 'application/json');
+          // hack: initiate a request to verify the backend API is working and
```

Comment:
Curious if there any alternatives you considered to this "hack"?

Reply:
@mgrbyte thanks for reviewing. What I consider a hack is to use the `/api/entity/` endpoint as a token validation endpoint. I imagine a `/api/myprofile` or `/api/person/me`  endpoint might appear more fitting, which is the reason I logged this #283.